seastar/src/timer.rs

```diff
@@ -7,3 +7,21 @@ mod ffi {
 }
 
 use ffi::*;
+
+fn caller<Func: FnMut()>(raw_func: *mut u8) {
+    unsafe { (raw_func as *mut Func).as_mut().unwrap()() }
+}
+
+const fn get_caller<Func: FnMut()>(_: &Func) -> fn(*mut u8) {
+    caller::<Func>
+}
+
+fn dropper<Func>(raw_func: *mut u8) {
+    unsafe {
+        let _ = Box::from_raw(raw_func as *mut Func);
+    }
+}
+
+const fn get_dropper<Func>(_: &Func) -> fn(*mut u8) {
+    dropper::<Func>
+}
```

Comment:
These should be put in the `ffi_utils` module, after this PR is merged: https://github.com/zpp-2022-rust-seastar/seastar-rs/pull/35

Reply:
Done. `caller` is called `fn_mut_void_caller` now. I don't think it's that good but it can't be called `fn_mut_caller` because it does not return any type like `fn_once_caller`.